src/content/docs/logs/forward-logs/cloudflare-logpush-forwarding.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: Forward logs using Cloudflare Logpush
+tags:
+  - Logs
+  - Enable log management in New Relic
+  - Enable log monitoring in New Relic
+  - Cloudflare
+  - Logpush
+metaDescription: "Configure Cloudflare's Logpush service to send logs to New Relic."
+---
+
+Configure the [Cloudflare Logpush](https://developers.cloudflare.com/logs/get-started/enable-destinations/new-relic/) service to send logs to New Relic using the Cloudflare dashboard or via API.
+
+## Manage via Cloudflare dashboard [#dashboard]
+
+To enable a Logpush service to New Relic via the Cloudflare dashboard:
+
+1.  Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login), and select the Enterprise domain you want to use with Logpush.
+
+2.  Go to **Analytics > Logs**.
+
+3.  Click **Connect a service** to open the modal window.
+
+4.  Select the dataset you want to push to a storage service.
+
+5.  Select the data fields to include in your logs. You can add or remove fields later by modifying your settings in **Logs > Logpush**.
+
+6.  Select **New Relic**.
+
+7.  Enter the **New Relic Logs Endpoint**:
+
+    US: `"https://log-api.newrelic.com/log/v1?Api-Key=<NR_LICENSE_KEY>&format=cloudflare"`
+
+    EU: `"https://log-api.eu.newrelic.com/log/v1?Api-Key=<NR_LICENSE_KEY>&format=cloudflare"`
+
+    Use the region that matches the one that has been set on your New Relic account. Replace `<NR_LICENSE_KEY>` with your New Relic license key. (You can [retrieve your license key](/docs/apis/intro-apis/new-relic-api-keys/#manage-license-key) from the UI or API.)
```

Comment:
I'd try to make this sentence easier: Use the region you set on your New Relic account.

Also, totally a personal thing, I'm not that fond of using (). For the last bit, what about "Replace `<NR_LICENSE_KEY>` with your New Relic license key, which [you can retrieve](/docs/apis/intro-apis/new-relic-api-keys/#manage-license-key) from the UI or API.